internal/cli/kraft/pkg/pkg.go

```diff
@@ -49,7 +49,7 @@ type PkgOptions struct {
 	NoKConfig    bool                      `local:"true" long:"no-kconfig" usage:"Do not include target .config as metadata"`
 	NoPull       bool                      `local:"true" long:"no-pull" usage:"Do not pull package dependencies before packaging"`
 	Output       string                    `local:"true" long:"output" short:"o" usage:"Save the package at the following output"`
-	Platform     string                    `local:"true" long:"plat" short:"p" usage:"Filter the creation of the package by platform of known targets"`
+	Platform     string                    `local:"true" long:"plat" short:"p" usage:"Filter the creation of the package by platform of known targets (qemu/fc/kraftcloud)" default:"qemu"`
```

Comment:
Same here, there is no clear default, so I would leave that out of the PR. The help text update is good though

Reply:
Thanks!
Already corrected.